src/main/frontend/handler/editor.cljs

```diff
@@ -828,10 +824,12 @@
                (when block-parent-id
                  (let [block-parent (gdom/getElement block-parent-id)
                        sibling-block (util/get-prev-block-non-collapsed-non-embed block-parent)
+                       before-editor-cursor (state/get-current-edit-block-and-position)
                        {:keys [prev-block new-content]} (move-to-prev-block repo sibling-block format id value)
                        concat-prev-block? (boolean (and prev-block new-content))
                        transact-opts (cond->
-                                       {:outliner-op :delete-block}
+                                       {:outliner-op :delete-block
+                                        :before-editor-cursor before-editor-cursor}
```

Comment:
Is there a reason why only this op gets :before-editor-cursor?

Reply:
The reason is that `move-to-prev-block` will move the cursor to the previous block, in the case of backspace at the beginning of a block, the expected undo behavior after `Backspace` is to put the cursor to the deleted block instead of the `prev` block.

I changed to not move the cursor for `delete-block` so `:before-editor-cursor` can be removed for this op.
 https://github.com/logseq/logseq/pull/9142/commits/2d381859a3eca4dd5041a1bb60fb590ef58930c3#diff-2f055ca9c790c38c4cbe601146ca7c19ad359f36a73554039581c2e5fae428d6R756